pages/bookmarks.jsx

```diff
@@ -21,14 +21,22 @@ export default React.createClass({
   handleLogOutBtnClick(event) {
     event.preventDefault();
     user.logout();
-    this.setState({ user });
   },
   componentDidMount() {
     // get IDs of user's bookmarked entries
     this.setState({bookmarks: getBookmarks()});
 
-    // let's verify what the logged in status is
-    user.verify(this.props.router.location, () => this.setState({ user }));
+    user.addListener(this);
```

Comment:
I wonder if I should import `app-user.js` as `User` (instead of `user`) since "user" is a class?